mitodl/micromasters
# added resource page and privacy policy page pre-population migration

#### Pre-Flight checklist


#### What are the relevant tickets?
Ticket #5025 

#### What's this PR do?
Adds privacy policy page.

#### How should this be manually tested?

Run migrations and visit /privacy-policy/
ensure page is visible and correct styles have been applied 

#### Screenshots (if appropriate)
Desktop Screen shots:

<img width="1440" alt="Screenshot 2021-09-03 at 10 17 08 AM" src="https://user-images.githubusercontent.com/87968618/131954339-81f018e5-cfc4-48fc-84eb-f526762f4d68.png">


Mobile screent shots

<img width="378" alt="Screenshot 2021-09-03 at 10 17 34 AM" src="https://user-images.githubusercontent.com/87968618/131954298-67d51500-162a-45d5-a0a6-fbfa2ab351bc.png">

<img width="378" alt="Screenshot 2021-09-03 at 10 18 05 AM" src="https://user-images.githubusercontent.com/87968618/131954316-d58e8560-0ab4-4a90-a74b-7c30cdd7d033.png">


CMS Page.

<img width="1440" alt="Screenshot 2021-09-07 at 10 24 59 AM" src="https://user-images.githubusercontent.com/87968618/132299492-77c4994c-8516-4893-a1b2-cff488fd60ee.png">

<img width="1440" alt="Screenshot 2021-09-07 at 10 25 15 AM" src="https://user-images.githubusercontent.com/87968618/132299517-4443b312-8f0d-48e6-a659-ff49f1631ac4.png">

<img width="1440" alt="Screenshot 2021-09-07 at 10 25 27 AM" src="https://user-images.githubusercontent.com/87968618/132299530-1a7ad411-d0ca-4ecc-9776-e671f103024f.png">

<img width="1440" alt="Screenshot 2021-09-07 at 10 25 45 AM" src="https://user-images.githubusercontent.com/87968618/132299553-5fecbdb2-22a9-4941-a158-3741d9ccf1af.png">

<img width="1440" alt="Screenshot 2021-09-07 at 10 25 54 AM" src="https://user-images.githubusercontent.com/87968618/132299568-3eae3f15-e076-4aeb-a8f1-f0f113f61c38.png">

Footer on page.

<img width="1440" alt="Screenshot 2021-09-07 at 11 28 29 AM" src="https://user-images.githubusercontent.com/87968618/132299645-e8df36c3-0038-45d2-abf3-06c8da214ad4.png">
